monarchmoney/monarchmoney.py

```diff
@@ -2352,14 +2392,54 @@ async def set_budget_amount(
             graphql_query=query,
         )
 
+    async def upload_and_parse_balance_history(
+        self,
+        account_id: str,
+        csv_content,
```

Comment:
needs a typehint

Reply:
To keep `upload_account_balance_history`, I've removed `upload_and_parse_balance_history` so this isn't necessary anymore.